drafts/linux-recommendations.md

```diff
@@ -0,0 +1,228 @@
+# Setup a secure Linux system
+
+## Why
+
+Free Software & Linux enable most of the [Status Principles](https://our.status.im/our-principles/) 
+
+Those principle include Security, Privacy and Censorship resistance.
+
+### 🛡️ Security 
+
+It is generally believed that Linux is a very secure operating system. 
+The security will depend largely on the distribution & setup. 
+
+Some Linux solutions distributions [have achieved a very high level of security standards & certifications](https://access.redhat.com/articles/2918071)
+
+### ㊙️ Privacy 
+
+Linux distributions have far greater respect for privacy than the competition from Microsoft, Google or Apple.
+Most Linux distributions will never require any form of identification to be installed, used or to access their app store.
+
+### 🤐 Censorship resistance 
+
+Apple, Google & Microsoft operating systems, along with their respective software distribution channels (App Store, Google Play & Microsoft Store) may censor specific applications. 
+
+This is much more difficult or even impossible on Linux.
+
+## How
+
+### Install Linux in a Virtual Machine (VM)
+
+Running Linux in a Virtual Machine is the best way to get a feel of what to expect.
+For some use cases, it could also be acceptable to operate long term by running Linux in a VM.
+But be aware from a security point of view, a VM and guest operating system will always somehow inherit the security of the host.
+Meaning that if your main host system is compromised, you should ultimately consider the guess insecure too. 
+
```

Comment:
guest compromise can be partially mitigated via disk encryption

Reply:
Sure, I added it